### PR TITLE
ready: add gradient overlay to photos

### PR DESCRIPTION
**Pull request by akhrail1996**

### CODE REVIEW

File: another/Feed/StoryPageCell.swift [hunk 1]
```diff
@@ -20,25 +38,20 @@ final class StoryPageCell: ASCellNode {
...
-//        if let backgroundImageNode = backgroundImageNode {
-            imageSpec = ASInsetLayoutSpec(insets: .zero, child: backgroundImageNode)
-//        } else {
-//            imageSpec = ASOverlayLayoutSpec()
-//        }
-        
-        return imageSpec
+        let imageSpec = ASInsetLayoutSpec(insets: .zero, child: backgroundImageNode)
+        let gradientSpec = ASInsetLayoutSpec(insets: .zero, child: gradientNode)
+        return ASOverlayLayoutSpec(child: imageSpec, overlay: gradientSpec)
```
Comment (ruslandoga) [inline another/Feed/StoryPageCell.swift:54]: no need to inset

```swift
let imageSpec = ASInsetLayoutSpec(insets: .zero, child: backgroundImageNode)
return ASOverlayLayoutSpec(child: imageSpec, overlay: gradientNode)
```

Comment (ruslandoga) [reply another/Feed/StoryPageCell.swift:54]: also grdient node should only be added to photos, what happens now?

**Review by ruslandoga — CHANGES_REQUESTED** [review]

File: another/Feed/StoryPageCell.swift [hunk 2]
```diff
@@ -5,6 +5,24 @@ final class StoryPageCell: ASCellNode {
...
+    private let gradientNode: GradientNode = {
+        let g = GradientNode(startingAt: CGPoint(x: 0.5, y: 0),
+                             endingAt: CGPoint(x: 0.5, y: 1),
+                             with: [UIColor.sinceBlack.withAlphaComponent(0.2),
+                                    UIColor.sinceBlack.withAlphaComponent(0),
+                                    UIColor.sinceBlack.withAlphaComponent(0),
+                                    UIColor.sinceBlack.withAlphaComponent(0.05),
+                                    UIColor.sinceBlack.withAlphaComponent(0.35)],
+                             for: [0.0,
+                                   0.15,
```
Comment (ruslandoga) [inline another/Feed/StoryPageCell.swift:17]: 0.15 is not enough to make hamburger button visible on pics with white bg, 0.25 seems to be enough though

File: another/Feed/StoryPageCell.swift (hunk 1 above)
Comment (akhrail1996) [reply another/Feed/StoryPageCell.swift:54]: it's just not there

**Review by ruslandoga — APPROVED** [review]

**Pull request merged**